### PR TITLE
Change structure

### PR DESCRIPTION
**Pull request by tadcka**
https://github.com/php-pds/skeleton

### CODE REVIEW

**Comment by andres-montanez** [comment]
Hi, thanks! I will take a look at the documentation, I didn't know this!

**Comment by tadcka** [comment]
Hi, okey :)

File: phpunit.xml.dist [hunk 1]
```diff
@@ -30,7 +30,7 @@
         <whitelist>
             <directory>./src/Mage/</directory>
```
Comment (andres-montanez) [inline phpunit.xml.dist]: Shouldn't this be `./src/` ?

**Comment by tadcka** [comment]
@andres-montanez done

**Comment by andres-montanez** [comment]
Awesome! I will merge it tonight.

**Pull request merged**